csrc/type.cpp

```diff
@@ -32,58 +32,6 @@ KernelIndexMode indexTypeToMode(DataType index_type) {
       : KernelIndexMode::INT64;
 }
 
-bool isFloatingPointType(DataType dtype) {
```

Comment:
This function is small, so I moved its definition to the header so that compiler can inline it.